config/7.1.0/securebanking/ig/scripts/groovy/RepoUser.groovy

```diff
@@ -14,13 +14,13 @@ userRequest.setMethod('GET');
 response = new Response(Status.OK)
 response.headers['Content-Type'] = "application/json"
 
-// split uri by filter request param
-def uriFilter = request.uri.toString().split("\\?")
-// split uri path to validate it
-def splitUriPath =  request.uri.path.split("/")
-
+// get uri query filter
+def uriQuery = new MutableUri(request.uri).getQuery()
```

Comment:
org.forgerock.http.protocol.Request#getUri is of type MutableUri, so you can just do request.uri.getQuery()

Reply:
True! fixed!